internal/config/mount_config.go

```diff
@@ -38,6 +38,13 @@ const (
 	// DefaultTypeCacheMaxEntries is the default vlaue of type-cache-max-entries.
 	// The value is set high at 1,048,476 (2^20) as it is not supposed to be set by users.
 	DefaultTypeCacheMaxEntries int = 2 << 20
+
+	// StatCacheMaxSizeMBUnsetSentinel is the value internally
+	// set for metada-cache:stat-cache-max-size-mb
+	// when it is not set in the gcsfuse mount config file.
+	// The constant value has been chosen deliberately
```

Comment:
i think we can be brief in comments unless not required to explain.  I would suggest to remove "The constant value has been chosen deliberately to be improbable for a user to explicitly set."

Reply:
done